product-plane-services/registry-server/src/main/java/org/opendatamesh/platform/pp/registry/server/database/entities/dataproductversion/DataProductVersion.java

```diff
@@ -64,48 +65,21 @@ public class DataProductVersion implements Cloneable, Serializable {
     @Column(name = "UPDATED_AT")
     private Date updatedAt;
 
-    /**
```

Comment:
This method has been moved in places where it is used, deleting hidden logic inside a jpa entity.